components/Head.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from 'react'
+import NextHead from 'next/head'
+
+type Props = {
```

Comment:
These can be expanded as needed. They were the only values that were set throughout the marketplace.